src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -68,5 +68,9 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
     <Compile Remove="corefx\System.Drawing\Perf_Graphics*.cs" />
   </ItemGroup>
+  
+    <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp3.0'">
```

Comment:
> This benchmark only runs on AVX2 and not in netcoreapp3.0? Is there a way to not bother building on non-x86* architecture?

@jorive it's compiled only for .NET Core 3.0. AFAIK there is no way to exclude it for given architecture (we target AnyCPU here, the target architecture depends on the dotnet cli we use)

Reply:
Does it build on arm*?

Reply:
yes

Reply:
The Avx2.IsSupported will just return false